cairo_programs/fibonacci.json

```diff
@@ -1,717 +1,684 @@
+              "end_col": 33,
+              "end_line": 17,
+              "input_file": {
+                "filename": "cairo_programs/fibonacci.cairo"
+              },
+              "start_col": 32,
+              "start_line": 17
+            },
+            "While expanding the reference 'y' in:"
+          ],
+          "start_col": 13,
+          "start_line": 16
+        }
+      },
+      "19": {
+        "accessible_scopes": ["__main__", "__main__.fib"],
+        "flow_tracking_data": {
+          "ap_tracking": {
+            "group": 2,
+            "offset": 3
+          },
+          "reference_ids": {
+            "__main__.fib.first_element": 1,
+            "__main__.fib.n": 3,
+            "__main__.fib.second_element": 2,
+            "__main__.fib.y": 5
+          }
         },
-        "__main__.main.ImplicitArgs": {
-            "full_name": "__main__.main.ImplicitArgs",
-            "members": {},
-            "size": 0,
-            "type": "struct"
+        "hints": [],
+        "inst": {
+          "end_col": 40,
+          "end_line": 17,
+          "input_file": {
+            "filename": "cairo_programs/fibonacci.cairo"
+          },
+          "start_col": 35,
+          "start_line": 17
+        }
+      },
+      "21": {
+        "accessible_scopes": ["__main__", "__main__.fib"],
+        "flow_tracking_data": {
+          "ap_tracking": {
+            "group": 2,
+            "offset": 4
+          },
+          "reference_ids": {
+            "__main__.fib.first_element": 1,
+            "__main__.fib.n": 3,
+            "__main__.fib.second_element": 2,
+            "__main__.fib.y": 5
+          }
         },
-        "__main__.main.Return": {
-            "cairo_type": "()",
-            "type": "type_definition"
+        "hints": [],
+        "inst": {
+          "end_col": 41,
+          "end_line": 17,
+          "input_file": {
+            "filename": "cairo_programs/fibonacci.cairo"
+          },
+          "start_col": 12,
+          "start_line": 17
+        }
+      },
+      "23": {
+        "accessible_scopes": ["__main__", "__main__.fib"],
+        "flow_tracking_data": {
+          "ap_tracking": {
+            "group": 3,
+            "offset": 0
+          },
+          "reference_ids": {
+            "__main__.fib.first_element": 1,
+            "__main__.fib.n": 3,
+            "__main__.fib.second_element": 2,
+            "__main__.fib.y": 5
+          }
         },
-        "__main__.main.SIZEOF_LOCALS": {
-            "type": "const",
-            "value": 0
+        "hints": [],
+        "inst": {
+          "end_col": 42,
+          "end_line": 17,
+          "input_file": {
+            "filename": "cairo_programs/fibonacci.cairo"
+          },
+          "start_col": 5,
+          "start_line": 17
+        }
+      }
+    }
+  },
+  "hints": {},
+  "identifiers": {
+    "__main__.fib": {
+      "decorators": [],
+      "pc": 11,
+      "type": "function",
+      "value": 11111111
```

Comment:
@jobez @lana-shanghai Here I had to put a random value as I had to do in other places because the parsing notes an error if there is no `value`. Here is the reference for this error during parsing,

https://github.com/lambdaclass/cairo-vm/blob/e6171d66a64146acc16d5512766ae91ae044f297/vm/src/serde/deserialize_program.rs#L479


Is there something I misunderstood, should we put a zero value here when there is nothing?

Reply:
kk, per discussion, this error is on the case where the identifier is a const, so we can undo these changes